### PR TITLE
[Brother] Add information about default password

### PR DESCRIPTION
**Pull request by Daniel15**
## Proposed change
Adding more details to the doc page for the Brother integration

## Type of change
- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
N/A

## Checklist
- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/brother.markdown [hunk 1]
```diff
@@ -32,7 +32,8 @@ Some very old Brother printers use different data format and these models are no
 
 ## Configuring the printer
 
-To enable SNMP, navigate to the printer's web interface (for example: `http://192.168.5.6`) and turn it on under Network / Protocol / SNMP.
+To enable SNMP, navigate to the printer's web interface (for example: `http://192.168.5.6`) and turn it on under Network / Protocol / SNMP. To see this option, you'll need to log in using the password field at the top of the page. For some printers, the default password is printed on a sticker on the back of the printer, preceded by "Pwd:". If the printer does not have a password on the sticker, the default password is "initpass".
```
Comment (bieniu) [inline source/_integrations/brother.markdown]:
```
To see this option, you'll need to log in using the password field at the top of the page.
...the default password is "initpass"
```
This is not true for all printer models. Some models do not have a password for the www panel by default.

**Review by bieniu — DISMISSED** [review]

File: source/_integrations/brother.markdown (hunk 1 above)
Comment (c0ffeeca7) [inline source/_integrations/brother.markdown]:
```suggestion
To enable SNMP, navigate to the printer's web interface (for example: `http://192.168.5.6`) and turn it on under Network / Protocol / SNMP. For some models, access to the web interface is password-protected. For some printers, the default password is printed on a sticker on the back of the printer, preceded by "Pwd:". If the printer does not have a password on the sticker, the default password is "initpass".
```
@bieniu, would this wording work?

Comment (bieniu) [reply source/_integrations/brother.markdown]: Oh my, I completely forgot about this PR. Yes, now it sounds good.

Comment (c0ffeeca7) [reply source/_integrations/brother.markdown]: No worries. Thank you for your feedback!

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @Daniel15 and @bieniu :+1:

**Comment by bieniu** [comment]
Thanks @c0ffeeca7

**Pull request merged**